aikg/utils/rdf.py

```diff
@@ -55,19 +55,25 @@
 # Retrieve each subject and its annotations
 SUBJECT_DOC_QUERY = """
 PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
-
-SELECT DISTINCT ?s ?sLab ?sCom
-WHERE
-{{
-    ?s rdfs:label ?sLab .
-    OPTIONAL {{
-        ?s rdfs:comment ?sCom .
-        ?o rdfs:label ?oLab .
-    }}
-    FILTER(LANG(?sLab) = "{lang}" || LANG(?sLab) = "")
-    FILTER(LANG(?sCom) = "{lang}" || LANG(?sLab) = "")
-    {graph_mask}
-}}
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+PREFIX schema: <http://schema.org/>
+SELECT DISTINCT ?s (SAMPLE(?sLab) as ?sLabel) ?sCom
+WHERE{
+    {
```

Comment:
```suggestion
WHERE
{{
```
**fix:** keep both curly braces (`{{`) on the same line to make f-string formater happy